src/export/event-exporter.service.js

```diff
@@ -36,12 +36,23 @@ function eventExporter(jobLogger, udbApi, EventExportJob) {
   ex.export = function (format, email, properties, perDay, customizations) {
     var queryString = ex.activeExport.query.queryString,
         selection = ex.activeExport.selection || [],
-        eventCount = ex.activeExport.eventCount;
+        eventCount = ex.activeExport.eventCount,
+        gaBrand = customizations.brand || '',
+        details = null,
+        user = $cookies.getObject('user');
 
     var jobPromise = udbApi.exportEvents(queryString, email, format, properties, perDay, selection, customizations);
+    if (_.get(appConfig, 'gaTagManager.containerId')) {
```

Comment:
Google specifieke code hoort hier niet thuis. @IbenTesara, zoals besproken op Slack splits je deze best af en decoreer je de export service in de app en overschrijf je daar de export methode om de nodige data mee te spelen om aan Google te geven.